conf/config.json

```diff
@@ -97,7 +97,48 @@
             "monitorReplicationFailureExpiryTimeS": 86400,
             "queueProcessor": {
                 "groupId": "backbeat-replication-group",
-                "retryTimeoutS": 300,
+                "retry": {
+                    "aws": {
+                        "maxRetries": 5,
+                        "timeoutS": 300,
+                        "backoff": {
+                            "min": 1000,
+                            "max": 30000,
+                            "jitter": 0.1,
+                            "factor": 1.5
+                        }
+                    },
+                    "azure": {
+                        "maxRetries": 5,
+                        "timeoutS": 300,
+                        "backoff": {
+                            "min": 1000,
+                            "max": 30000,
+                            "jitter": 0.1,
+                            "factor": 1.5
+                        }
+                    },
+                    "gcp": {
+                        "maxRetries": 5,
+                        "timeoutS": 300,
+                        "backoff": {
+                            "min": 1000,
+                            "max": 30000,
+                            "jitter": 0.1,
+                            "factor": 1.5
+                        }
+                    },
+                    "default": {
```

Comment:
Looks like this will also apply to a Scality RING destination because there is no `site` property. Thinking aloud here: maybe we could move the default values to the Joi config validator (i.e. using the `default()` method) and then rename this something like "scality".